HTML + CSS + Js files

dolahimself opened the pull request:
This goes into the HTML document of a file via react Js DOM manipulation, to add a message field to a log of messages.

babatopeoni commented on DOM.js:
```diff
@@ -0,0 +1,26 @@
+document.addEventListener('DOMContentLoaded', function(){
```
There is a single responsibility principle that states a method should do only one thing. Your event listener is doing too many things. Create other methods to handle different parts. Your event listener should just call one method. create other methods to do other parts

dolahimself committed:
HTML + CSS + Js files

This ges into the html document of a file via react Js DOM manipulation, to add a message field to a log of messages.

dolahimself committed:
Update DOM.js

Decongesting the block of Js code, for every method to have a single responsibility. It's yet to give me my desired result, I'm still in progress.

Line 27 returns error.

dolahimself committed:
Update DOM.js
